service/src/test/java/uk/gov/hmcts/reform/fpl/service/ApplicationDocumentsServiceTest.java

```diff
@@ -0,0 +1,171 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.jackson.JacksonAutoConfiguration;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.fpl.model.ApplicationDocument;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.common.DocumentReference;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.service.time.Time;
+import uk.gov.hmcts.reform.fpl.utils.DocumentUploadHelper;
+import uk.gov.hmcts.reform.fpl.utils.FixedTimeConfiguration;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.fpl.utils.CoreCaseDataStoreLoader.caseData;
+import static uk.gov.hmcts.reform.fpl.utils.CoreCaseDataStoreLoader.emptyCaseData;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.element;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = {
+    JacksonAutoConfiguration.class,
+    ApplicationDocumentsService.class,
+    FixedTimeConfiguration.class
+})
+class ApplicationDocumentsServiceTest {
```

Comment:
I would say it would be nice to remove the dependancy on spring by things like so
```java 
    private final Time time = new FixedTimeConfiguration().stoppedTime();
    private final ObjectMapper mapper = new ObjectMapper();
    private final DocumentUploadHelper documentUploadHelper = mock(DocumentUploadHelper.class);
    private final ApplicationDocumentsService applicationDocumentsService = new ApplicationDocumentsService(time, documentUploadHelper);
```
however due to your use of
```
import static uk.gov.hmcts.reform.fpl.utils.CoreCaseDataStoreLoader.caseData;
import static uk.gov.hmcts.reform.fpl.utils.CoreCaseDataStoreLoader.emptyCaseData;
```
this would require a large amount of change to get this to work

Reply:
sure, leaving for now as close to closing pr